src/components/warning-callout/WarningCallout.tsx

```diff
@@ -1,14 +1,26 @@
 import React, { HTMLProps } from 'react';
 import classNames from 'classnames';
 
-const WarningCallout: React.FC<HTMLProps<HTMLDivElement>> = ({
+interface WarningCalloutProps extends HTMLProps<HTMLDivElement> {
+  labelProps?: HTMLProps<HTMLHeadingElement>;
+}
+
+const WarningCallout: React.FC<WarningCalloutProps> = ({
   className,
   label,
   children,
+  labelProps,
   ...rest
 }) => (
-  <div className={classNames('nhsuk-warning-callout')} {...rest}>
-    {label ? <h3 className="nhsuk-warning-callout__label">{label}</h3> : null}
+  <div className={classNames('nhsuk-warning-callout', className)} {...rest}>
+    {label ? (
+      <h3
+        className={classNames('nhsuk-warning-callout__label', labelProps?.className)}
```

Comment:
Might want to think about doing this a different way, currently if you pass in className as a labelProp, then it will override the existing className rather than merging with it. Unless that is the expected behaviour, in which case we don't need to use the classNames() helper here

Reply:
Ah yeah - that's a good point, as the className in the labelProps would still be in there even though it's used, good catch!!